firmware/shiftx3_api.h

```diff
@@ -69,9 +83,9 @@ struct ConfigGroup1 {
 };
 
 /* API offsets */
-#define SHIFTX2_CAN_BASE_ID     0xE3600
-#define SHIFTX2_CAN_API_RANGE   256
-#define SHIFTX2_CAN_FILTER_MASK 0x1FFFFF00
+#define SHIFTX3_CAN_BASE_ID     0xE3600
```

Comment:
Would it make sense to change this so that a ShiftX3 can be added to a system that already has 2 ShiftX2 devices?  Is this something that happens?

Reply:
Since it's very similar to ShiftX2 (just has the 7 segment LED and an extra switch) I'd like to keep the API and control as close as possible to make the front end configuration easier. 

Also, I think it's a pretty unlikely situation and if they do, they could update the jumpers to offset either the ShiftX2 or X3.  Good point though!